test/Python/Schedule.lf

```diff
@@ -0,0 +1,23 @@
+// Example from Schedule section of the C Reactor Target wiki page.
+target Python;
+reactor Schedule {
+    input x;
+    logical action a;
+    reaction(x) -> a {=
+        a.schedule(MSEC(200))
```

Comment:
We could rename this function to something like `msec`. The native Python types are really not that great for our purposes. For example, `datetime` has a resolution of microseconds and `time` works with floating point seconds. There are relatively new nanosecond resolution functionality such as `time.clock_gettime_ns` since Python 3.7 (more [here](https://vstinner.github.io/python37-pep-564-nanoseconds.html)). However, that would limit our compatibility. I think we still need our own functions.

Reply:
I think the implementation is fine, I'm just commenting on the fact that the functions are written in uppercase. I think they should be lowercase.